lib/_internal/utils/conditions.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:fhir/r4.dart';
+import 'package:prapare/_internal/constants/snomed_enum.dart';
+
+Condition getCondition(SNOMED snomed, Id id) => Condition(
+      subject: Reference(reference: 'Patient/$id'),
+      clinicalStatus: CodeableConcept(
+        coding: [
+          Coding(
+            system: FhirUri(
+                'http://terminology.hl7.org/CodeSystem/condition-clinical'),
+            code: Code('active'),
+            display: 'Active',
+          ),
+        ],
+      ),
+      verificationStatus: CodeableConcept(
+        coding: [
+          Coding(
+              system: FhirUri(
+                  'http://terminology.hl7.org/CodeSystem/condition-ver-status'),
+              code: Code('confirmed'),
+              display: 'Confirmed')
+        ],
+      ),
+      category: [
+        CodeableConcept(
+          coding: [
+            Coding(
+              system: FhirUri(
+                  'http://hl7.org/fhir/us/core/CodeSystem/condition-category'),
+              code: Code('health-concern'),
+            )
+          ],
+        )
+      ],
+      code: sdohConcept[snomed],
+      onsetPeriod: Period(
+        start: FhirDateTime(
+          DateTime(
+            DateTime.now().year - 1,
```

Comment:
Calling `DateTime.now()` repeatedly can be optimized to a single stored value…otherwise you’ll have minor discrepancies if you needed millisec accuracy. Also, since you call this multiple times, a util function may be helpful.